### PR TITLE
Add video blog post 10 for April 14th

### PR DESCRIPTION
**Pull request by mgienow**
Daniel Kim on teaching beginner webdevs using Gatsby

### CODE REVIEW

**Comment by laurieontech** [comment]
Awesome!

File: docs/blog/2020-04-14-LA-2020-Kim/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: 'Gatsby Days LA 2020 Video 10: Teaching Web Development to Beginners with Gatsby'
+date: 2020-04-14
+author: Greg Thomas
+excerpt: "UC Davis grad student Daniel Kim explains why Gatsby offers
+a better framework than React for teaching beginning web development
+at Gatsby Days LA 2020."
+tags:
+- gatsby-days
```
Comment (muescha) [inline docs/blog/2020-04-14-LA-2020-Kim/index.md:9]: missing spaces?

**Comment by muescha** [comment]
blogposts LA Days higher than Video 7 are not visible in https://www.gatsbyjs.org/blog/ and in https://www.gatsbyjs.org/blog/tags/gatsby-days

**Comment by muescha** [comment]
filed issue:

- #23147 `bug(blogs): blog posts are not visible`

File: docs/blog/2020-04-14-LA-2020-Kim/index.md (hunk 1 above)
Comment (muescha) [reply docs/blog/2020-04-14-LA-2020-Kim/index.md:9]: opened a PR:
- #23148 `fix(blog): LA Days Kim - frontmatter`

**Pull request merged**